ci/container-images-tag-and-push.sh

```diff
@@ -26,13 +28,32 @@ if [ -n "${REGISTRY_PREFIX}" ]; then
     fi
 fi
 
-docker tag ${ZEEK_IMAGE_REPO}/zeek-multiarch:arm64 ${REGISTRY_PREFIX}${ZEEK_IMAGE_REPO}/${ARCH_IMAGE_NAME}:${ARCH_IMAGE_TAG}-arm64
-docker tag ${ZEEK_IMAGE_REPO}/zeek-multiarch:amd64 ${REGISTRY_PREFIX}${ZEEK_IMAGE_REPO}/${ARCH_IMAGE_NAME}:${ARCH_IMAGE_TAG}-amd64
-docker push ${REGISTRY_PREFIX}${ZEEK_IMAGE_REPO}/${ARCH_IMAGE_NAME}:${ARCH_IMAGE_TAG}-arm64
-docker push ${REGISTRY_PREFIX}${ZEEK_IMAGE_REPO}/${ARCH_IMAGE_NAME}:${ARCH_IMAGE_TAG}-amd64
+# Forward arguments to docker and retry the command once if failing (e.g network issues).
+function do_docker {
+    if ! docker "$@"; then
+        docker "$@"
```

Comment:
I'm sure you considered this, but should this sleep before the retry?

Reply:
Ah yes, was thinking more of the upload stops after a few seconds and stalls and immediate retry will be success. I'll add 5 seconds, but otherwise no advanced retry mechanisms - there's task retry on a higher level, too.